test/controllers/processors/AppealStorageFormSubmissionProcessor.test.ts

```diff
@@ -24,7 +24,7 @@ describe('AppealStorageForSubmissionProcessor', () => {
     const appeal: Appeal = {
         penaltyIdentifier: {
             companyNumber: '00345567',
-            penaltyReference: 'A00000001',
+            userInputPenaltyReference: 'A00000001',
```

Comment:
what's the reason for changing this? only the `penaltyReference` is saved in mongo?

Reply:
My bad, didn't pay attention